mern-ecommerce/routes/order.js

```diff
@@ -37,7 +37,7 @@ router.put(
   requireSignin,
   isAuth,
   isAdmin,
-  updateOrderStatus
+  updateOrder
```

Comment:
The function `updateOrder` is being used in the router.put method, but it's not defined in this file. It's actually `updateOrderStatus` that's imported from the `../controllers/order` module. This is a mismatch and can cause issues. It should be `updateOrderStatus` instead of `updateOrder`.